backend/src/main/java/woowacourse/touroot/travelogue/dto/TraveloguePlaceRequest.java

```diff
@@ -0,0 +1,31 @@
+package woowacourse.touroot.travelogue.dto;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotNull;
+import java.util.List;
+import woowacourse.touroot.place.domain.Place;
+import woowacourse.touroot.travelogue.domain.day.domain.TravelogueDay;
+import woowacourse.touroot.travelogue.domain.place.domain.TraveloguePlace;
+
+public record TraveloguePlaceRequest(
+        @Schema(description = "여행기 장소 이름", example = "선릉 캠퍼스")
+        @NotNull(message = "여행기 장소 이름은 비어있을 수 없습니다.")
+        String name,
+        @Schema(description = "여행기 장소 위치 정보")
+        @NotNull(message = "여행기 장소 위치 정보는 비어있을 수 없습니다.")
+        TravelogueLocationRequest location,
+        @Schema(description = "여행기 장소 설명", example = "성담 빌딩에 위치한 선릉 캠퍼스입니다.")
+        String description,
+        @Schema(description = "여행기 장소 사진")
+        @NotNull(message = "여행기 장소 사진은 비어있을 수 없습니다.")
+        List<TraveloguePhotoRequest> photos
```

Comment:
어노테이션이 많아져서 한 눈에 안 들어오는데 변수마다 개행을 넣어주는건 어떻게 생각하시나요?

```suggestion
        @Schema(description = "여행기 장소 이름", example = "선릉 캠퍼스")
        @NotNull(message = "여행기 장소 이름은 비어있을 수 없습니다.")
        String name,
        
        @Schema(description = "여행기 장소 위치 정보")
        @NotNull(message = "여행기 장소 위치 정보는 비어있을 수 없습니다.")
        TravelogueLocationRequest location,
        
        @Schema(description = "여행기 장소 설명", example = "성담 빌딩에 위치한 선릉 캠퍼스입니다.")
        String description,
        
        @Schema(description = "여행기 장소 사진")
        @NotNull(message = "여행기 장소 사진은 비어있을 수 없습니다.")
        List<TraveloguePhotoRequest> photos
```

Reply:
`record` 파라미터를 개행해도 무방할까요? 보통 파라미터는 개행하질 않아서 이렇게 작성했습니다!